### ISSUE TITLE
Visually differentiate suspended tabs

### ISSUE THREAD

**Issue by mohanraj-r** [deanoemcke/thegreatdiscarder#6]
@deanoemcke Heard about the [great discarder extension when you mentioned it in a issue](https://github.com/deanoemcke/thegreatsuspender/issues/512#issuecomment-306694878) - I just installed and tried it. Looks like a great alternative to the great suspender. The only thing I miss so far is the visual faded-out distinction of suspended tabs. Is it not possible with the great discarder ?

Also have you considered merging the two extensions into one and providing an option to the user to choose the suspend backend ? It would help take advantage of the existing large user base of the great suspender and offer those users a great alternative. I didn't know about this extension till now - from your comment. 

Thanks for all your hard work on these great extensions.

**Comment by deanoemcke** [comment]
Unfortunately, a the appearance and behaviour of a discarded tab is completely out of my control. 

I did consider trying to merge the two concepts, but ultimately I felt like they were too different to be combined as one. For example, if you choose tab discarding as your suspension strategy, a lot of the options and features of the extension would no longer be applicable and would cause confusion (ie. your favicon issue).

Also, by ditching a lot of the unneccessary code from The Great Suspender, I was able to make this extension much leaner. And require much less permissions.

**Comment by mohanraj-r** [comment]
Thanks for the explanation. Makes sense. 

Maybe it would be useful to cross-reference the extensions in their chrome webstore pages if they are not already ?

**Comment by mohanraj-r** [comment]
@deanoemcke Would it be possible to add some info e.g. the list of suspended tabs (in maybe the extension settings) or count of suspended tabs / total tabs (on the extension icon) - to give a sense of how many tabs have been suspended / how well the extension is working. Because right now I have no way of knowing this except to click each tab and see if its reloading or launch the chrome task manager and compare tabs to processes.

**Comment by deanoemcke** [comment]
Yes. I agree that it would be nice to have a bit more visibility on which tabs are currently discarded. In the meantime, you chrome provides an internal way to access this information: `chrome://discards`

**Comment by deanoemcke** [comment]
Just thinking a bit more about this and the best UI for it. 

@mohanraj-r In regards to your pull request, do you think it will be obvious for people that the number represents the number of discarded tabs? Is this information that we constantly need to see as an overlay? I'm interested in hearing your use-case for wanting this information.

I think there could be a mutual interest here with the pull request from @jeffmahoney to close all discarded tabs: https://github.com/deanoemcke/thegreatdiscarder/pull/3
As I've already mentioned to him in this mirror PR on The Great Suspender (https://github.com/deanoemcke/thegreatsuspender/pull/499) I am reluctant to add this to the popup menu as I don't feel the use-case for it is popular enough.

Could we potentially show a summary of discarded tabs somewhere else - perhaps another view accessible from the sidebar in the options screen, that both shows how many tabs are discarded, and also has a button to close them all? It could also have discard all / reload all buttons, as well as the ability to manually discard / reload each individual tab.
Maybe I'm going overboard here. It's a lot more work for sure. Keen to hear your thoughts.

**Comment by mohanraj-r** [comment]
> Could we potentially show a summary of discarded tabs somewhere else - perhaps another view accessible from the sidebar in the options screen, that both shows how many tabs are discarded, and also has a button to close them all? It could also have discard all / reload all buttons, as well as the ability to manually discard / reload each individual tab.

👍 I was thinking along the same lines about another page (in options e.g.) for showing the list of discard tabs, maybe even list the tabs about to be discarded with a remaining time to discard. I just went with the badge count since it seemed like the easier thing to do (I am a newbie to JS and extensions dev) - with the plan of following up with another view. 

I will give it a try when I can get some time in a week or so.

**Comment by ExSport** [comment]
@deanoemcke Very nice extension 👍 
Showing total number of tabs and nr. of suspended ones will be nice feature as you will get info about active ones like 142/32 meaning 142 tabs, 32 suspended so there is a chance to free some memory by manual suspends or by reconfiguring time in settings etc...
It will be also nice to integrate info from chrome://discards (in extension status/report page or only redirecting to it for further info about the state of discarding...for people with no knowledge about this hidden page)
Thx!

**Comment by ExSport** [comment]
@deanoemcke Not sure if to post is as bug or feature but spotted that "Discard OTHER tabs" discards only tabs in active window, not all running windows.
If it is not bug than I am asking for new feature to "Discard ALL tabs" 👍 
Many thanks!

**Comment by deanoemcke** [comment]
@ExSport This is a feature :) (the discarding only tabs in current window).
Could you please raise this request in a separate issue? Thanks!

**Comment by drone540** [comment]
What about using the ideas from this extension to fix this problem?

https://github.com/rNeomy/auto-tab-discard

This file is where they call the discard and set the icon
https://github.com/rNeomy/auto-tab-discard/blob/master/background.js
